attributevalue/src/main/scala/com/hiya/alternator/syntax/package.scala

```diff
@@ -19,5 +19,9 @@ package object syntax {
     def between[T : ScalarDynamoFormat](lower: T, upper: T): RKCondition.BETWEEN[T] = RKCondition.BETWEEN(lower, upper)
   }
 
-
+  object cond {
+    def attr[T](name: String): ConditionExpression.Path[T] = ConditionExpression.Attr(name)
+    def lit[T: DynamoFormat](literal: T): ConditionExpression[T] = ConditionExpression.Literal(literal)
+    def not(expr: ConditionExpression[Boolean]): ConditionExpression[Boolean] = expr.not
```

Comment:
I'd rather use unary_! in ConditionExpression. It needs a bit of magic to check that it is a boolean, though.